Taskfile.yml

```diff
@@ -220,32 +220,29 @@ tasks:
 
   ci:pa11y:
     desc: Run Pa11y to test accessiblity
-    deps: [ci:pa11y:pre]
+    deps: [ci:prepare]
     cmds:
       - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm -e HTTP_PROXY={{ .WIREMOCK_HOST_DOCKER }} node-chrome yarn pa11y:ci
 
-  ci:pa11y:browser:
+  ci:wiremock:browser:
     desc: Watch the site with a wiremock http proxy. ⚠️  Depends on Chrome installed on OS ⚠️
-    deps: [ci:pa11y:browser:warning, ci:pa11y:pre]
+    deps: [ci:prepare]
     cmds:
-      - yarn pa11y:browser
+      - echo '⚠️  Please make sure that Chrome is installed in your OS.'
+      - yarn wiremock:browser
     env:
       HTTP_PROXY: '{{ .WIREMOCK_HOST_LOCAL }}'
 
-  ci:pa11y:browser:warning:
-    cmds:
-      - echo '⚠️  Please make sure that Chrome is installed in your OS.'
-
-  ci:pa11y:pre:
-    cmds:
-      - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm node-chrome yarn install --frozen-lockfile
-      - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm -e HTTP_PROXY={{ .WIREMOCK_HOST_DOCKER }} node-chrome yarn pa11y:create-mappings
-
   ci:lighthouse:
     desc: Run Lighthouse to test performance
+    deps: [ci:prepare]
+    cmds:
+      - docker compose {{ .DOCKER_COMPOSE_FILES_CI }} run --rm -e HTTP_PROXY={{ .WIREMOCK_HOST_DOCKER }} node-chrome npx lhci autorun
+
+  ci:prepare:
```

Comment:
Consider renaming this task and provide a description.

`ci:prepare` is very generic and sounds like something we would setup for all `ci:*` tasks, but we actually only do so for Pa11y and Lighthouse. It is (currently) not something we use with e.g. Cypress.

Reply:
Yeah. Naming is hard. I choose the generic version because I couldn't come up with a better alternative.

Reply:
Found  a better name - I guess :)

ci:wiremock:create-mappings